src/js/index.js

```diff
@@ -0,0 +1,73 @@
+import { EditorState } from '@codemirror/state';
+import { EditorView } from '@codemirror/view';
+import { basicSetup } from 'codemirror';
+import JSONFormatter from 'json-formatter-js';
+import '../styles/styles.css';
+import convertStringToObject from './utils/convertStringToObject';
+import getStateDifference from './utils/getStateDifference';
+import { Zamok } from './zamok';
+
+const zamok = new Zamok.api({
+  topWindowOrigin: 'http://localhost:8081/?zamok=16.2.0',
```

Comment:
I'm concerned about the version number being in here.  It's fine for now, because we don't release to Production very often.  We have a Collection in Firestore called `defaultVersions`.  I think in the future we might make a server-side call to get the current version.  For now, this is fine, but it might fall out of sync in the future.